roles/ovirt-host-deploy-libvirt-guests/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Fetch info about VDSM
+  command: bash -c "rpm -qi vdsm | grep -oE 'Version\\s+:\\s+[0-9\\.]+' | awk '{print $3}'"
+  changed_when: false
+  register: vdsm_version
+
+- block:
+    - name: Change default timeout if setup
+      ini_file:
+        state: present
+        section: null
+        path: /etc/sysconfig/libvirt-guests
+        option: "SHUTDOWN_TIMEOUT"
+        value: "{{ host_deploy_shutdown_timeout }}"
+      when:
+        - host_deploy_shutdown_timeout is defined
+
+    - name: Change run in parallel if setup
+      ini_file:
+        state: present
+        section: null
+        path: /etc/sysconfig/libvirt-guests
+        option: "PARALLEL_SHUTDOWN "
+        value: "{{ host_deploy_shutdown_parallel }}"
+      when:
+        - host_deploy_shutdown_parallel is defined
+
+    - name: Start libvirt-guests service
+      service:
```

Comment:
Is this restart save? Should we restart it only when configuration is changed?

Reply:
It is actually never supposed to restart. I only need to ensure it is started and enabled. That is all.
If the service is restarted it will perform is shutdown event and will shutdown all registered VMs. 

This behaviour is given by its implementation:
- the libvirt-guests is old init-v script ported to work in systemd, by very simple method of making it oneshot command executed on every registered event [startup, shutdown, restart, suspend]
- everytime libvirt-guests is fired-up it reloads the config file and performs its registered actions
- shutdown is trigered on systemd stop
- start is trigered on systemd start
- shutdown is triggered on systemd restart

Reply:
@petrkotas, so then you do not want to stop and start it while some VMs are running during this task, do you?

Reply:
But still if you change the configuration, shouldn't you restart the service? And if there are no changes just make sure service is running and don't restart it?

Reply:
@machacekondra you do not need to, that is the beauty of the old script. The configuration gets loaded every time the script gets executed, it does not run as daemon.

@michalskrivanek of course not, therefore you do not restart it and do not touch it until absolutely necessary. This is the reason, why you start it and do not restart.